cert/aziot-certd/src/config.rs

```diff
@@ -349,8 +359,36 @@ aziot_certd = "unix:///var/run/aziot/certd.sock"
 			].into_iter().collect(),
 
 			endpoints: super::Endpoints {
-				aziot_certd: http_common::Connector::new(&"unix:///var/run/aziot/certd.sock".parse().unwrap()).unwrap(),
-				aziot_keyd: http_common::Connector::new(&"unix:///var/run/aziot/keyd.sock".parse().unwrap()).unwrap(),
+				aziot_certd: http_common::Connector::Fd {
+					fallback: Box::new(http_common::Connector::Unix { socket_path: std::path::Path::new("/run/aziot/certd.sock").into() }),
+				},
+				aziot_keyd: http_common::Connector::Unix { socket_path: std::path::Path::new("/run/aziot/keyd.sock").into() },
```

Comment:
Where do we `set_permissions` on `/run/aziot` for IS/KS/CS groups?

Reply:
That's systemd's job. The `/run/foo` directories are `r-x` to world, and the socket files themselves are ACLed based on the `SocketUser` and `SocketGroup` of the individual socket unit.

```sh
$ ls -l /run/ | grep aziot

drwxr-xr-x.  2 root root   60 Oct  9 00:26 aziot


$ ls -l /run/aziot/

total 0
srw-rw----. 1 aziotcs aziotcs 0 Oct  9 11:29 certd.sock
srw-rw----. 1 aziotid aziotid 0 Oct  9 11:29 identityd.sock
srw-rw----. 1 aziotks aziotks 0 Oct  9 11:29 keyd.sock
```